### DISCUSSION TITLE
Thanks for the new util!

### DISCUSSION THREAD

**Discussion by SnowDogDad** [rodneyviana/ODSyncUtil#6] [General]
Hi Rodney, thanks for developing this utility. My organization desperately needs something to help us ensure our user's are actually logging in to OneDrive. 

I put this on my Windows 11 desktop (fully updated) at home where I have three instances of OneDrive running. 

My question is very generic.  What should I see when ODSyncUtil.exe runs? The PS1 generates an error saying the $output variable is null. When I run the exe by itself, I see nothing. It just returns to a command prompt.

**Comment by rodneyviana** [comment]
You should see like is in the description. You need to run in the context of the user. Only the OneDrives related to the user should show. Did you unblock the files after downloading? Don’t run as administrator. If it can’t find a driver, the output of the exe should be [] (empty array). Returning nothing does not make any sense.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Jim McBee ***@***.***>
Sent: Saturday, February 10, 2024 10:02:42 PM
To: rodneyviana/ODSyncUtil ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [rodneyviana/ODSyncUtil] Thanks for the new util! (Discussion #6)


Hi Rodney, thanks for developing this utility. My organization desperately needs something to help us ensure our user's are actually logging in to OneDrive.

I put this on my Windows 11 desktop (fully updated) at home where I have three instances of OneDrive running.

My question is very generic. What should I see when ODSyncUtil.exe runs? The PS1 generates an error saying the $output variable is null. When I run the exe by itself, I see nothing. It just returns to a command prompt.

**Reply by SnowDogDad** [reply]
I am running it in a PowerShell window within my user context. I did unblock the file. 

I think it is probably not finding the driver it is looking for.  I *thought* was running the latest Win 11 Pro build, but I just checked and it is 22H2. I'll update to 23H2 and then do some more testing. 

Thanks for the response!

**Comment by rodneyviana** [comment]
Keep me posted

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
